nixnet/system/databases.py

```diff
@@ -0,0 +1,147 @@
+﻿from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import itertools
+import six
+import typing  # NOQA: F401
+
+from nixnet import _funcs
+
+
+class Databases(collections.Mapping):
+    """Database aliases."""
+
+    def __init__(self, handle):
+        self._handle = handle
+
+    def __repr__(self):
+        return 'System.Databases(handle={0})'.format(self._handle)
+
+    def __len__(self):
+        return len(_funcs.nxdb_get_database_list(''))
+
+    def __iter__(self):
+        for alias, filepath in _funcs.nxdb_get_database_list(''):
```

Comment:
`for alias, _ in ...`

`_` is a generally recommended placeholder for "don't care" values